code/modules/vending/autodrobe.dm

```diff
@@ -133,6 +133,7 @@
 				   /obj/item/shield/riot/roman/fake = 1,
 				   /obj/item/clothing/suit/chaplainsuit/clownpriest = 1,
 				   /obj/item/clothing/head/clownmitre = 1,
+				   /obj/item/reagent_containers/food/snacks/fakefruit/skull = 1,
```

Comment:
why is there food in the clothing machine?

Reply:
It's not technically to be food; it's a fake prop skull.

It's subclass of food so it can be ground up into sillicon.